.github/workflows/review-assigner.yaml

```
@@ -0,0 +1,41 @@
+name: "Reviewer Assigner"
+
+on:
+    pull_request:
+        types: [opened]
+    issues:
+        types: [opened]
+
+jobs:
+  create-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@e69ef5462fd455e02edcaf4dd7708eda96b9eda0 # v7.0.0
+        with:
+          script: |
+                console.log("triggered by: " + context.eventName)
+                // console.log("Assigning to <insert from pagerduty here>")
+                 if (context.eventName == "issues"){
+                     try{
+                         await github.rest.issues.addAssignees({
+                             owner: context.repo.owner,
+                             repo: context.repo.repo,
+                             assignees: ["BBBmau"], // this will be replaced with pagerduty resp
+                             issue_number: context.issue.number, // issue.number is used for both issues and PRs
+                         })
+                     } catch(error){
+                         core.setFailed("error assigning assignee:" + error)
```

The repetition sounds funny. Maybe rephrase with "reviewer" ?
```suggestion
                         core.setFailed("error assigning reviewer:" + error)
```